core/services/relay/evm/read/event.go

```diff
@@ -665,9 +716,10 @@ func wrapInternalErr(err error) error {
 
 	errStr := err.Error()
 	if strings.Contains(errStr, "not found") || strings.Contains(errStr, "no rows") {
-		return fmt.Errorf("%w: %w", commontypes.ErrNotFound, err)
+		return fmt.Errorf("%w: %s", commontypes.ErrNotFound, err.Error())
```

Comment:
Why are these being undone? Including both errors should be strictly better.
```suggestion
		return fmt.Errorf("%w: %w", commontypes.ErrNotFound, err)
```